python/activity_stream/resources/note_widget.ui

```diff
@@ -155,9 +167,6 @@
         <property name="wordWrap">
          <bool>true</bool>
         </property>
-        <property name="textInteractionFlags">
```

Comment:
I am having a little bit of a hard time reading the xml, so maybe totally wrong here :) - but are we removing the ability to click on links and select text here? If so, what's the reason?